README.md

```diff
@@ -1,31 +1,95 @@
-# perseo
+# Perseo Context-Aware CEP
 
-[![FIWARE Processing](https://nexus.lab.fiware.org/static/badges/chapters/processing.svg)](https://www.fiware.org/developers/catalogue/)
-[![License: APGL](https://img.shields.io/github/license/telefonicaid/perseo-fe.svg)](https://opensource.org/licenses/AGPL-3.0)
-<br/>
-[![Build badge](https://img.shields.io/travis/telefonicaid/perseo-fe.svg)](https://travis-ci.org/telefonicaid/perseo-fe/)
-[![Coverage Status](https://coveralls.io/repos/github/telefonicaid/perseo-fe/badge.svg?branch=master)](https://coveralls.io/github/telefonicaid/perseo-fe?branch=master)
+[![FIWARE Processing, analysis and visualization of context information](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/visualization.svg)](https://www.fiware.org/developers/catalogue/)
+[![License: AGPLv3+](https://img.shields.io/badge/License-AGPLv3+-blue.svg)](./LICENSE)
+[![Docker Status](https://img.shields.io/docker/pulls/fiware/perseo.svg)](https://hub.docker.com/r/fiware/perseo/)
+[![Support badge](https://img.shields.io/badge/tag-fiware--perseo-orange.svg?logo=stackoverflow)](https://stackoverflow.com/questions/tagged/fiware-perseo)
+<br> [![Documentation badge](https://img.shields.io/readthedocs/perseo.svg)](https://perseo.readthedocs.io/en/latest/)
```

Comment:
Is https://perseo.readthedocs.io/en/latest the final site for RTD doc?

I'm asking as the "while we finish" make me think it is temporal and probably the PR should point to the final one (although at this moment it appears a broken)

> While we finish with this PR, the documentation can be found here:
> https://perseo.readthedocs.io/en/latest/

Reply:
We just did not have very clear if you already had a readthedocs for perseo (https://readthedocs.org/projects/fiware-perseo-fe/ and https://readthedocs.org/projects/fiware-perseo-core/). 
If I remember correctly, we come to the conclusion of having only one domain for documentation for Perseo and not separating it into 'core' and 'fe', I can give you access to [this new one](https://perseo.readthedocs.io/en/latest) readthedocs so that you can manage it and keep this new url. ok?

Reply:
https://perseo.readthedocs.io/en/latest is ok.

NTC